cli/CacheWarmingCommand.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Cli;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Finder\Finder;
+use Twig\Error\Error;
+use Twig\Loader\FilesystemLoader;
+use WMDE\Fundraising\Frontend\Factories\FunFunFactory;
+
+class CacheWarmingCommand extends Command {
+
+	private const NAME = 'app:warm-cache';
+
+	private FunFunFactory $funFunFactory;
+
+	public function __construct( FunFunFactory $funFunFactory ) {
+		parent::__construct( self::NAME );
+		$this->funFunFactory = $funFunFactory;
+		$this->funFunFactory->setLocale( 'de_DE' );
+	}
+
+	protected function configure(): void {
+		$this->setName( self::NAME )->setDescription( 'Generate Twig cache' );
+	}
+
+	protected function execute( InputInterface $input, OutputInterface $output ): int {
+		try {
+			foreach ( $this->getTemplates() as $template ) {
+				$this->funFunFactory->getSkinTwig()->createTemplate( $template );
+			}
+		}
+		catch ( Error $e ) {
+			return Command::FAILURE;
+		}
+
+		return Command::SUCCESS;
+	}
+
+	private function getTemplates(): \ArrayIterator {
+		$loader = $this->funFunFactory->getSkinTwig()->getLoader();
+		if ( !$loader instanceof FilesystemLoader ) {
+			throw new \InvalidArgumentException( 'Expected FilesystemLoader.' );
+		}
+
+		$templates = [];
+		foreach ( $loader->getNamespaces() as $namespace ) {
+			foreach ( $loader->getPaths( $namespace ) as $path ) {
+				$templates = array_merge( $templates, $this->findTemplatesInDirectory( $path, $namespace ) );
+			}
+		}
+
+		return new \ArrayIterator( array_unique( $templates ) );
+	}
+
+	private function findTemplatesInDirectory( string $dir, string $namespace ): array {
```

Comment:
```suggestion
       /**
        * @return string[]
        */
	private function findTemplatesInDirectory( string $dir, string $namespace ): array {
```

For future-proof support of higher PHPStan levels

```